### ISSUE TITLE
Nuget?

### ISSUE THREAD

**Issue by Shadowblitz16** [simplexidev/sdfx#36]
**Is your feature request related to a problem? Please describe.**
Kinda. can you upload this to nuget?

**Describe the solution you'd like**
Upload as .net 5 nuget package

**Describe alternatives you've considered**
I have considered Eto.Forms but its too complicated.

**Additional Information**
Please separate stable and unstable releases

**Comment by Shadowblitz16** [comment]
No Its ok. I just really want a good cross platform native gui.
Are there any plans on allowing basic drawing and audio?

**Comment by tom-corwin** [comment]
The real plan for this project is to use it as a separate repository to get a half-decent API for `libui` before it goes into a project, that has yet to actually have a set name yet, allowing for UI, Audio (via miniaudio), and Drawing (maybe opengl, maybe not), and then some. More information about that will come in the future, though.

**Comment by Shadowblitz16** [comment]
I don't know how opengl could draw on controls, maybe it would use different graphics behind the scenes?

**Comment by tom-corwin** [comment]
Let me think on it tonight while I get the repo (and CI) fixed up for .NET 6. 

By the morning, there will be a separate issue pertaining to audio and graphics, respectively. Once work begins on those issues, the repo will get a rename (which I'll open an issue for suggestions when the time comes).

P.S. Thanks for showing interest in this project, as I may have completely forgotten about it and realized it needs a major face lift :slight_smile:

**Comment by Shadowblitz16** [comment]
I would contribute but I have yet to even make a asteroids clone in C# with a game engine so I think I am pretty unsuited for contribution.

Also I always mess up git when I use it DX.

**Comment by tom-corwin** [comment]
Honestly, my only background in UI of any kind before this project, was WinForms. Graphics is gonna be a fun one.

And as for the git mess ups, have you tried a UI for it like [GitKraken](https://gitkraken.com)? I use it as a daily driver, and it does anything I've ever needed to do with git.

**Comment by Shadowblitz16** [comment]
I have not no I usually use the terminal or built in gui within my ide.

**Comment by tom-corwin** [comment]
#39 *should* build as-is, for .NET 6. No promises or warranties, though!

**Comment by tom-corwin** [comment]
Depends on #28

**Comment by tom-corwin** [comment]
I have most of the work done for this locally, but GitHub has disabled my account's package feed, so once they get back to me, the first CI builds should be available on GitHub through my account's NuGet feed.

**Comment by simplexidev** [comment]
I'd like to correct that last comment, as I used MyGet instead. You can look for the packages, but keep in mind the only usable one will be `LibUISharp.Core`, as I have *not* uploaded the binaries for the `libui`, `miniaudio`, or `zstd`.
